lib/adapters/http/http.js

```diff
@@ -10,11 +10,21 @@ var CHANGES_BATCH_SIZE = 25;
 var MAX_URL_LENGTH = 1800;
 
 var utils = require('../../utils');
+var Promise = utils.Promise;
+var base64 = require('../../deps/base64');
+var btoa = base64.btoa;
+var atob = base64.atob;
```

Comment:
I'm trying to get in the habit of moving dependencies up top, because 1) it makes it easier to extract them from `utils` later, and 2) uglify can shorten the variable names.